.drone.yml

```diff
@@ -653,6 +653,60 @@ steps:
       - e2e/cloud-plugins-suite/azure-monitor.spec.ts
     repo:
     - grafana/grafana
+- commands:
+  - sleep 10s
+  - yarn e2e:playwright
+  depends_on:
+  - grafana-server
+  environment:
+    HOST: grafana-server
+    PORT: "3001"
+    PROV_DIR: /grafana/scripts/grafana-server/tmp/conf/provisioning
+  failure: ignore
```

Comment:
Q: shouldn't we fail the pipeline if this step fails?

Reply:
Good catch - should have made this clear in the comment above...yes we should eventually! Just want to see this step run in the main branch for a week or so first. Want to be certain that these tests don't introduce any kind of flakiness.

Reply:
Make sense 👍🏻